src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -5,12 +5,38 @@ title: JavaScript
 ## Integrating the SDK
 All our JavaScript-related SDKs provide the same API. Still, there are some differences between them, such as installation, which this section of the docs explains.
 
-{% include components/platform_content.html content_dir='getting-started-install' %}
+The quickest way to get started is to use the CDN hosted version of the JavaScript browser SDK:
+
+``` javascript
+<script src="https://browser.sentry-cdn.com/5.4.0/bundle.min.js" crossorigin="anonymous"></script>
+```
+
+You can also add the Sentry SDK as a dependency using yarn or npm:
+
+``` bash
+# Using yarn
```

Comment:
can we only show one? people who use yarn know how to use yarn. also "npm save" is the equiv to "yarn add"

Reply:
`npm install --save-prod` etc. But apparently `install` saves now too: https://stackoverflow.com/a/19578808/1544347